bodhi/templates/override.html

```diff
@@ -60,7 +61,7 @@ <h2 class="pull-left">New Override</span>
         </div>
       </div>
 
-      <div class="col-md-6">
+      <div class="col-md-3">
```

Comment:
Looks okay here:

![override](https://cloud.githubusercontent.com/assets/331338/9742415/f54d53b8-562d-11e5-8986-d5007cfefb9a.png)

I guess it could use a little padding.. but so could the other panels.